src/components/People.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import './People.css';
+
+const BASE_URL = 'https://ghibliapi.herokuapp.com/people';
+
+const PeoplePage = () => {
+  const [search, setSearch] = useState('');
+  const [person, setPerson] = useState(null);
+  const [notFound, setNotFound] = useState(false);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+
+    const response = await fetch(`${BASE_URL}?name=${search}`);
```

Comment:
you would need to use `/people` endpoint and then find the person based on the user input